content/specific-skill-success-criteria/classes-and-objects/_index.md

```diff
@@ -23,3 +23,118 @@ All students need to provably understand all of the following concepts:
 - getters and setters
 - inheritance and overriding and extending methods
 - composition
+
+# Python
+
+## Constructors
+The `__init__()` method is called the constructor and is always called when an object is created, generally used for instantiating an object.
```

Comment:
That phrasing sounds a bit odd, I would say:
The `__init__()` method is called the constructor and is always called when an object is created - also known as instantiating an object.